src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipItem.qml

```diff
@@ -243,22 +252,51 @@ Rectangle {
                 id: buttonsRow
 
                 anchors.right: parent.right
-                anchors.rightMargin: 4
+                anchors.rightMargin: root.rightVisibleMargin + 4
```

Comment:
I'm just gonna leave a comment here so we don't forget to clarify what should happen with hiding icons beyond right edge of the screen.
Not mentioning anybody as discussion is handled somewhere else I see

Reply:
dropped the commit with this feature